amti/CMakeLists.txt

```diff
@@ -38,27 +38,34 @@ if(ENABLE_amtiplatforms)
 
     yarp_install(FILES amtiplatforms.ini  DESTINATION ${YARP_PLUGIN_MANIFESTS_INSTALL_DIR})
 
-    YARP_PREPARE_PLUGIN(amtiforceplate TYPE yarp::dev::AMTIForcePlate
-                                       INCLUDE include/AMTIForcePlate.h
-			               CATEGORY device
-				       EXTRA_CONFIG WRAPPER=AnalogServer)
+	YARP_PREPARE_PLUGIN(amtiforceplate TYPE yarp::dev::AMTIForcePlate
+                                   INCLUDE AMTIForcePlate.h
+								   CATEGORY device
+								   EXTRA_CONFIG WRAPPER=AnalogServer)
     if(ENABLE_amtiforceplate)
 
+        add_compile_definitions(_USE_MATH_DEFINES) #For using M_PI macro
+        find_package(iDynTree REQUIRED)
+        find_package(Eigen3 REQUIRED)

```

Comment:
> since we multiply a wrench, we should use the inverse of the adjoint matrix

What you mean here?

Reply:
In any case, I agree in general that iDynTree is an overkill for this simple rotation. YARP or Eigen are indeed a good idea to use.

Reply:
> > since we multiply a wrench, we should use the inverse of the adjoint matrix
> 
> What you mean here?

I meant we should use `adjointInv(.)` here, am I right?

Reply:
> > > since we multiply a wrench, we should use the inverse of the adjoint matrix
> > 
> > 
> > What you mean here?
> 
> I meant we should use `adjointInv(.)` here, am I right?

ok probably not, I just saw how it is defined in the documentation since the transpose is missing, then I think we should generate the wrench adjoint transform  as:
```cpp
yarp::sig::Matrix m_adjointTransformWrench = yarp::sig::Matrix::transposed( yarp::math::adjointInv(m_transform));
```

Reply:
Ok!

Reply:
@lrapetti that is a great catch! Thanks 😉 
Corrected and tested in https://github.com/robotology/forcetorque-yarp-devices/commit/85140f0181a83a8589250ed425ba0ec78a61b40a

![Screenshot 2020-10-16 140727](https://user-images.githubusercontent.com/6505998/96256712-6f8fe600-0fb9-11eb-8c8e-17bd99e6bd67.png)

Reply:
Great thanks!
I think you just missed to remove the `find_package` at https://github.com/robotology/forcetorque-yarp-devices/pull/20/files#diff-a2016d7dc9a1f17e43e64ca2bbfda668505c78274d6da523032f2b2fb6ece69eR48-R49 then we can merge it

Reply:
Done in https://github.com/robotology/forcetorque-yarp-devices/pull/20/commits/415dba22c19b8c7ffa89b597377fee698557b994